### ISSUE TITLE
'Convert to Fahrenheit' button is in enable state when no text is entered in the edit field.

### ISSUE THREAD

**Issue by ayushjai19** [microsoft/WinUI-Gallery#2085]
### Which version of the app?

WinUI 3 Gallery

### Description

Environment Details:
OS Version: Dev (OS Build 27881.1000)
App: WinUI 3 Gallery
App Version: 2.6.0.0 (SDK 1.7)

Repro Steps:
Launch the WinUI 3 Gallery app.
Navigate to search box using tab key.
Type 'Custom &user controls' in search box and hit enter.
Navigate to 'Custom &user controls' card and activate it.
Navigate to "User Control" heading.
Observe ''Convert to Fahrenheit' button.
Actual Result:
'Convert to Fahrenheit' button is in enable state when no text is entered in the edit field. 

Expected Result:
'Convert to Fahrenheit' button should be in disabled state when no text is entered in the edit field. 

Page source link: https://github.com/microsoft/WinUI-Gallery/blob/main/WinUIGallery/Samples/ControlPages/Fundamentals/CustomUserControlsPage.xaml

### Windows version

Windows 11 24H2 (26100)
